frontend/src/services/OrderService.ts

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+import {Order} from "../types/Order.ts";
+
+export default class OrderService {
+    getAllOrders() {
+        return axios.get('/api/orders');
```

Comment:
in my opinion, outsourcing only the "axios.get()" to a seperate class doesn't make much sense. 
Either also include the 
`.then((response) => {
            setOrders(response.data);
            setOrdersByStatus(response.data);`
            in your "getAllOrders" function, to clean up your useEffect in your OrderList.tsx. Or just write the entire "getAllOrders" in your OrderList.tsx. 
            In my opinion, outsourcing the axios requests to a seperate "OrderService.ts" file doesn't make much sense, since it's not a lot of code that you safe, if you rather write down the axios requests in the respective frontend components. This way, you can also better and easier see, what you're actually fetching with the axios request, when reading the code in your components

Reply:
I think defining axios operations in a separate class is a better use of object-oriented programming and SOLID.